src/angular-sql-query.js

```diff
@@ -65,25 +111,21 @@
     }
 
     /**
-     * Request a specific entry by his id
-     *
-     * @param  {Object} entryId - Id of the entry to request
-     * @return {Promise}       - Request result
-     * @this SqlQueryService
+     * @param  {string}                entryId -
+     * @return {ng.IPromise<Resource>}         -
+     * @this   {SqlQuery}
      */
     function getBackUp(entryId) {
       if(!entryId) {
-        throw new Error({ err: 'We need an id' });
+        throw new Error('FiltersParameters need filter');
```

Comment:
I don't think that this error will be understandable when we use the library.
This error exists to mention to the user if he misses passing the id.

Reply:
fixed